test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/ControllerSpec.scala

```diff
@@ -131,15 +133,15 @@ trait ControllerSpec extends AnyWordSpec with Matchers with BeforeAndAfterAll wi
     super.afterAll()
   }
 
-  def messageFromMessageKey(messageKey: String, args: Any*)(implicit messagesApi: MessagesApi): String = {
-    val m = messagesApi(messageKey, args: _*)(lang)
+  def messageFromMessageKey(messageKey: String, args: Any*): String = {
```

Comment:
why did you change this?

Reply:
Why retrieve it in every testcase and then pass it in, when it's already available in the same file?

Reply:
Rather write extra 5 lines In those 2-3 test cases we want to test welsh, than doing it in every single test case